CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+Version 1.12.0
+==============
+
+This release provides a number of exciting new features as well as a set
+of important bug fixes. Many thanks to our users for reporting these
+issues and @poplarShift @reckoner @jonmease for their code contributions
+and @scottclowe @syhooper for their contributions to the documentation.
```

Comment:
All the ands and the lack of commas made this a bit ugly so maybe just merge them together:

>Many thanks to our users for reporting these issues, and special thanks to @poplarShift, @reckoner, @jonmease, @scottclowe and @syhooper for their contributions to the code and the documentation.

Reply:
Fixed in df58ab544